src/Models/AlbumsModel.php

```diff
@@ -0,0 +1,54 @@
+<?php
+class AlbumsModel {
+    public PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    public function getAlbumById(int $id): Album
+    {
+        $query = $this->db->prepare('SELECT `albums`.`id` AS "albumId", `albums`.`album_name` AS "albumName",  
+        `albums`.`artwork_url` AS "artworkURL", `albums`.`artist_id` AS "artistId" FROM `albums` WHERE `albums`.`id` = :id;');
+        $query->setFetchMode(PDO::FETCH_CLASS, Album::class);
+        $query->execute(['id' => $id]);
+        return $query->fetch();
+    }
+
+    public function getAllAlbums(): array
+    {
+        $query = $this->db->prepare('SELECT `albums`.`id` AS "albumId", `albums`.`album_name` AS "albumName",  
+        `albums`.`artwork_url` AS "artworkURL", `albums`.`artist_id` AS "artistId" FROM `albums`;');
+        $query->setFetchMode(PDO::FETCH_CLASS, Album::class);
+        $query->execute();
+        return $query->fetchAll();
+    }
+
+    public function getAlbumsByArtistId(int $artistId): array
```

Comment:
You should add the return comments to these methods to get phpstorm autocompletions when using arrays of artists